benchmarks/install_and_test.sh

```diff
@@ -75,6 +76,11 @@ function runJavaBenchmark(){
   ./gradlew :benchmarks:run --args="-resultsFile \"${BENCH_FOLDER}/$1\" --dataSize \"$2\" --concurrentTasks \"$concurrentTasks\" --clients \"$chosenClients\" --host $host $portFlag --clientCount \"$clientCount\" $tlsFlag $clusterFlag $minimalFlag"
 }
 
+function runGoBenchmark(){
+    cd ${BENCH_FOLDER}/../go/benchmarks
+    go run . -resultsFile ${BENCH_FOLDER}/$1 -dataSize $2 -concurrentTasks $concurrentTasks -clients $chosenClients -host $host $portFlag -clientCount $clientCount $tlsFlag $clusterFlag $minimalFlag
```

Comment:
Is there a way to have options prefixed with double dash to comply with other benchmarks (and common standard)?